impress-org/givewp
# Errors when connecting to Stripe using PHP 8 are now resolved

Resolves #5873

## Description

This PR resolves the issue where connecting to Stripe using PHP 8 on the server was causing errors. The workaround was to downgrade the version of PHP to PHP 7.4. 

The issue is resolved by using the latest version of the `stripe/stripe-php` package.

Tested on a server with various PHP versions:

- PHP 5.6.39
- PHP 7.3.5
- PHP 8

## Testing Instructions

1. Make sure you are using `PHP 8` on your server and that `WP_DEBUG` is enabled
2. Connect the stripe account
3. Make a donation

## Pre-review Checklist
